terraform/accounts/main/route53.tf

```diff
@@ -40,3 +40,14 @@ resource "aws_route53_record" "ci_marketplace_team" {
     "${var.jenkins_ip}",
   ]
 }
+
+resource "aws_route53_record" "ci2_marketplace_team" {
```

Comment:
Is this subdomain redundant now? Is the plan to point ci.marketpalce.team at a new Jenkins box? If so, should the record the A record above be pointed at the new boxes ip instead?

Or maybe it should be a two phase thing - bring the new box up, then switch the record when we're happy its healthy.

Reply:
Until this works then we need to run both servers in parallel. So this PR can land while we're still using the current Jenkins.

Reply:
Cool. Didn't you mention in slack the other day that you were deleting the ci2. domain?

Reply:
Yeah I was checking that if I wiped it all, I could get it back again :)